Gulpfile.js

```diff
@@ -1,38 +1,40 @@
-var gulp = require( 'gulp' );
-var sass = require( 'gulp-sass' );
-var rename = require( 'gulp-rename' );
-var cssmin = require( 'gulp-cssmin' );
-var uglify = require( 'gulp-uglify' );
-var livereload = require( 'gulp-livereload' );
+const gulp = require( 'gulp' );
+const sass = require('gulp-sass')(require('sass'));
+const rename = require( 'gulp-rename' );
+const cleanCSS = require('gulp-clean-css');
+const uglify = require( 'gulp-uglify' );
+const livereload = require( 'gulp-livereload' );
 
-gulp.task( 'sass', function() {
+function compileSass() {
 	return gulp.src( 'assets/scss/**/*.scss' )
 		.pipe( sass().on( 'error', sass.logError ) )
-		.pipe( gulp.dest( 'assets/css' ) )
-		.pipe( livereload() );
-} );
+		.pipe( gulp.dest( 'assets/css' ) );
+}
 
-gulp.task( 'css', [ 'sass' ], function() {
-	return gulp.src( [ 'assets/css/**/*.css', '!assets/css/**/*.min.css' ] )
-		.pipe( rename( { suffix: '.min' } ) )
-		.pipe( cssmin() )
-		.pipe( gulp.dest( 'assets/css' ) )
-} );
+// Seems like cleanCSS is
+function cssMin() {
+    return gulp.src( ['assets/css/**/*.css', '!assets/css/**/*.min.css'] )
+        .pipe(rename({ extname: '.min.css' }))
+        .pipe( cleanCSS() )
+        .pipe( gulp.dest( 'assets/css' ) )
+}
 
-gulp.task( 'js', function() {
+function js() {
 	return gulp.src( [ 'assets/js/**/*.js', '!assets/js/**/*.min.js' ] )
 		.pipe( uglify( {
 			mangle: false
 		} ) )
 		.pipe( rename( { suffix: '.min' } ) )
 		.pipe( gulp.dest( 'assets/js' ) )
 		.pipe( livereload() );
-} );
+};
 
-gulp.task( 'default', [ 'css', 'js' ] );
-
-gulp.task( 'watch', [ 'css', 'js' ], function() {
+function runLivereload() {
 	livereload.listen();
-	gulp.watch( [ 'assets/scss/**/*.scss' ], [ 'css' ] );
-	gulp.watch( [ 'assets/js/**/*.js' ], [ 'js' ] );
-} );
\ No newline at end of file
+	gulp.watch( [ 'assets/scss/**/*.scss' ], css );
```

Comment:
@rosswintle this is failing. I believe this needs to be changed from `css` to `sass`.

Reply:
I was wrong. That should be `compileSass`. Then you should be good.

Reply:
I made a better fix. And the `watch` job now works properly too. Thanks!